app/views/user_mailer/in_person_ready_to_verify.html.erb

```diff
@@ -76,7 +76,7 @@
 
 <% if @presenter.selected_location_details.present? %>
   <div class="margin-y-4">
-    <h2 class="font-sans-md margin-bottom-1 text-normal text-bold"><%= @presenter.selected_location_details['name'] %></h2>
+    <h2 class="font-sans-md margin-bottom-1 text-bold"><%= @presenter.selected_location_details['name'] %></h2>
```

Comment:
Per the reference Figma, this is intended to be bolded. Removing `text-normal` allows us to avoid adding it to the stylesheet.